<launch>

  <!-- launch the experiment controller node -->
  <node name="experiment_controller" pkg="bwi_exp1" type="experiment.py">
    <param name="graph_file" value="$(find bwi_exp1)/config/graph.yaml" />
    <param name="experiment_file" value="$(find bwi_exp1)/config/experiment.yaml" />
  </node>

  <node name="robot_positioner" pkg="bwi_exp1" type="robot_positioner">
    <param name="map_file" value="$(find utexas_gdc)/maps/3ne.yaml" />
    <param name="graph_file" value="$(find bwi_exp1)/config/graph.yaml" />
    <param name="debug" value="true" />
  </node>

</launch>
